manta-parameters/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "manta-parameters"
+version = "0.5.0"
+edition = "2021"
+authors = ["Manta Network <contact@manta.network>"]
+readme = "README.md"
+license-file = "LICENSE"
+repository = "https://github.com/Manta-Network/manta-rs"
+homepage = "https://github.com/Manta-Network"
+documentation = "https://github.com/Manta-Network/manta-rs"
+categories = [""]
+keywords = [""]
+description = "Concrete Parameters and Interfaces for Manta."
+publish = false
+
+[package.metadata.docs.rs]
+# To build locally:
+# RUSTDOCFLAGS="--cfg doc_cfg" cargo +nightly doc --all-features --open
+all-features = true
+rustdoc-args = ["--cfg", "doc_cfg"]
+
+[badges]
+is-it-maintained-issue-resolution = { repository = "Manta-Network/manta-rs" }
+is-it-maintained-open-issues = { repository = "Manta-Network/manta-rs" }
+maintenance = { status = "actively-developed" }
+
+[features]
+# Enable Standard Library
+std = []
+
+# Download Data from GitHub
+download = ["anyhow/std", "attohttpc", "std"]
```

Comment:
In new Rust, we can write this as:

```suggestion
# Download Data from GitHub
download = ["anyhow", "attohttpc", "std"]

# Enable Standard Library
std = ["anyhow?/std"]
```

and let's have the features sorted too.